.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Cucumber",
```

Comment:
Because that's what it's for in VSCode. Launching and debugging from within the IDE.